### PR TITLE
Remove cnpy dependency for example-talos-arm

### PR DESCRIPTION
**Pull request by ManifoldFR**
The relevant code is not that useful; this should fix the conda-forge recipe [build error](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=864628&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823)

@jorisv @olivier-roussel

### CODE REVIEW

**Comment by jorisv** [comment]
Add a changelog entry in Changed section: Don't output numpy matrices in example-talos-arm.

**Comment by ManifoldFR** [comment]
> Add a changelog entry in Changed section: Don't output numpy matrices in example-talos-arm.

Done!

**Review by jorisv — APPROVED** [review]

File: examples/talos-arm.cpp [hunk 1]
```diff
@@ -40,8 +22,6 @@ int main(int, char **) {
   double mu_init = 0.01;
   SolverProxDDP<double> solver(TOL, mu_init, 0., max_iters, aligator::VERBOSE);
   std::string fpath_base = "kkt_matrices";
```
Comment (olivier-roussel) [inline examples/talos-arm.cpp:24]: Variable is not used anymore

Comment (olivier-roussel) [reply examples/talos-arm.cpp:24]: Yep, not a big deal though...

**Pull request merged**